src/SearchParameters/index.js

```diff
@@ -1593,7 +1629,7 @@ SearchParameters.prototype = {
     var parsedParams = SearchParameters._parseNumbers(params);
 
     return this.mutateMe(function mergeWith(newInstance) {
-      var ks = keys(params);
+      var ks = Object.keys(params);
```

Comment:
Since global `keys` doesn't exist anymore, we can rename this variable to `keys`.